src/meta/geo/forest.ts

```diff
@@ -101,6 +101,7 @@ export const sourcesMetadata = {
     scale: 231.6563,
     palette: ['#FFD700'], // gold
     citation: 'https://lpdaac.usgs.gov/products/mod44bv006/',
+    forestAreaDataProperty: 'undefined',
```

Comment:
this looks weird. can forestAreaDataProperty be optional in its type definition?

Reply:
I added a check before using the property, so now this assignment is not necessary.